perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -1639,6 +1639,48 @@ public void getResourcesSpecifiedByVoAndFacilityWhereUserIsNotAdminButHisGroupIs
 		assertTrue(resources.contains(resource));
 	}
 
+	@Test
+	public void getResourcesSpecifiedByVoWhereUserIsAdmin() throws Exception {
+		System.out.println(CLASS_NAME + "getResourcesSpecifiedByVoWhereUserIsAdmin");
+
+		vo = setUpVo();
+		member = setUpMember(vo);
+		facility = setUpFacility();
+		Facility facility2 = new Facility();
+		facility2.setName("ResourcesManagerTestFacility2");
+		facility2 = perun.getFacilitiesManager().createFacility(sess, facility2);
+		resource = setUpResource();
+		Resource resource2 = new Resource();
+		resource2.setName("name");
+		resource2 = resourcesManager.createResource(sess, resource2, vo, facility2);
+		User u = perun.getUsersManagerBl().getUserByMember(sess, member);
+
+		resourcesManager.addAdmin(sess, resource, u);
+		resourcesManager.addAdmin(sess, resource2, u);
+		List<Resource> resources = resourcesManager.getResourcesWhereUserIsAdmin(sess, vo, u);
+
+		assertNotNull(resources);
+		assertTrue(resources.contains(resource));
+		assertTrue(resources.contains(resource2));
```

Comment:
Nit: This can be done easily with assertJ:
`assertThat(resources).containsOnly(resource, resource2);`
This is the import you need for the method:
`import static org.assertj.core.api.Assertions.assertThat;`